source/core/slang-performance-profiler.cpp

```diff
@@ -1,5 +1,7 @@
 #include "slang-performance-profiler.h"
 #include "slang-dictionary.h"
+#include <atomic>
```

Comment:
Do you still need to include `atomic`?
I am not sure which one on your current code uses it.

Reply:
done.